doc/ref_arch/openstack/chapters/chapter06.md

```diff
@@ -127,31 +127,31 @@ The following rules govern create, read, update, and delete (CRUD) level access.
 <a name="6.3.3"></a>
 ### 6.3.3 Confidentiality and Integrity
 
-Confidentiality implies that data and resources must be protected against malicious attacks such as introspection and Integrity implies that the data must be protected from unauthorized modifications or deletions.
+Confidentiality implies that data and resources must be protected against exfiltration. Integrity implies that the data must be protected from unauthorized modifications or deletions.
```

Comment:
@karinesevilla Confidentiality doesn't always result in data being transferred out -- it also covers unauthorised reading of data that was covered by introspection.

Reply:
@pgoyal01 If a 3rd party reads a datum or data, they've implicitly exfiltrated it. It doesn't matter if they use cat or they use ftp to effect the exfiltration, the data is out of the authorized domain.

Reply:
@pgoyal01 @markshostak The nuance between both is small. There is no issue to keep both of them. I will modify the text